src/components/Table/components/Row.tsx

```diff
@@ -123,6 +124,13 @@ function RowImpl<R extends Kinded, S>(props: RowProps<R>): ReactElement {
       [` > .${revealOnRowHoverClass} > *`]: Css.vh.$,
       [`:hover > .${revealOnRowHoverClass} > *`]: Css.vv.$,
     },
+    ...{
+      [`.${editableOnRowHoverClass} .textFieldBaseWrapper`]: Css.pl1.mlPx(-8).br4.ba.bcTransparent.bgTransparent.$,
+      [`.${editableOnRowHoverClass} .textFieldBaseWrapper > *`]: Css.bgTransparent.$,
+      [`.${editableOnRowHoverClass}:hover .textFieldBaseWrapper`]: Css.bgBlue100.$,
```

Comment:
@0ces cc @apattersonATX-HB (sorry, I'm just getting to this PR) this is a ~fairly intrusive way to "reach into" the text field and muck with its internal state / styling / etc...

Like it gets the job done, but a maintainer of `TextField` would have no idea this behavior is happening, and could unwittingly break it while updating/maintaining `TextField`.

I would generally/greatly prefer some sort of "signal" from GridTable to `TextField` to tell it "please style yourself (this way/that way)".

Granted, it would have to be a context, given we want to cross component boundaries / we're not passing props directly to `TextField`.

My off-the-top-of-my-head suggestion would be to use `PresentationContext`, which is already how GridTable will broadcast to child/cell components "btw you're in a table,k you should render slightly differently".

Can we add something like that, for this ask / hopefully in this PR (it seems like a quick thing to try/experiment with)? Like a ... `PresentationContext.visuallyEditable`? That would kinda match the current `visuallyDisabled` prop...

Reply:
Actually it looks like GridTable is already setting `PresentationContext.borderless: true`, to tell components "hide your borders", and it seems like the real ask here is tweaking/adding a `borderless: "until-hovered"` sort of thing ...

Or maybe having the GridTable flip `borderless: true / false` based on when the row itself is hovered?

Granted, context changes mean a React re-render to change the flag from true/false, instead of just doing the `:hover` ... but I think that's probably fine?